frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx

```diff
@@ -14,5 +17,12 @@ export const getTopologyShortcuts = (t: TFunction): React.ReactElement => (
     <Shortcut ctrl keyName="Spacebar">
       {t('topology~Open quick search modal')}
     </Shortcut>
+    {options?.extensions?.length > 0 && (
+      <Shortcut customTitle={t('topology~Drag + Drop')}>
+        {t('topology~Upload file ({{fileTypes}}) to project', {
```

Comment:
shouldn't we pluralize `file` if more than one type of file extension is supported?

Reply:
I didn't pluralize this is because I don't want the user to get the impression that he/she can upload multiple files with a single drag-n-drop 
@lwrigh please let me know what you think

Reply:
@debsmita1 I would agree to keep it singular for the time being since the user can only upload one file at a time currently. We can always make it plural if we end up supporting uploading multiple files at the same time in the future.